### ISSUE TITLE
not working? or i have wrong settings

### ISSUE THREAD

**Issue by cannap** [marhali/easy-i18n#46]
hi i have a project with a locales folder but i will not recognize the files  they are json but i also need to keep the js files they just export the json but itlooks like its not working  
![image](https://user-images.githubusercontent.com/1394151/129881846-b2fe1364-4968-4acf-90b8-84143b33d175.png)

thanks

**Comment by cannap** [comment]
the js files just do this 

```js
import validations from 'vee-validate/dist/locale/de.json'
import de from './de.json'
const customValidations = {}

export default {
  ...de,
  validations: { ...validations.messages, ...customValidations }
}

```

**Comment by cannap** [comment]
ok when the folder includes other files then it will break i putted all my json files in a json folder now its working

**Comment by marhali** [comment]
You can use the "Translation file pattern" configuration to only select the json files. Try something like `*.json`

**Comment by cannap** [comment]
i tried this gives

```
java.util.regex.PatternSyntaxException: Dangling meta character '*' near index 0
*.json
^
	at java.base/java.util.regex.Pattern.error(Pattern.java:2027)
	at java.base/java.util.regex.Pattern.sequence(Pattern.java:2202)
	at java.base/java.util.regex.Pattern.expr(Pattern.java:2068)
	at java.base/java.util.regex.Pattern.compile(Pattern.java:1782)
	at java.base/java.util.regex.Pattern.<init>(Pattern.java:1428)
	at java.base/java.util.regex.Pattern.compile(Pattern.java:1068)
	at java.base/java.util.regex.Pattern.matches(Pattern.java:1173)
	at java.base/java.lang.String.matches(String.java:2024)
	at de.marhali.easyi18n.util.IOUtil.isFileRelevant(IOUtil.java:68)
	at de.marhali.easyi18n.io.implementation.ModularizedJsonTranslatorIO.lambda$read$0(ModularizedJsonTranslatorIO.java:59)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:828)
	at de.marhali.easyi18n.io.implementation.ModularizedJsonTranslatorIO.read(ModularizedJsonTranslatorIO.java:39)
	at de.marhali.easyi18n.service.DataStore.reloadFromDisk(DataStore.java:82)
	at de.marhali.easyi18n.dialog.SettingsDialog.showAndHandle(SettingsDialog.java:49)
	at de.marhali.easyi18n.action.SettingsAction.actionPerformed(SettingsAction.java:24)
	at com.intellij.openapi.actionSystem.impl.ActionButton.actionPerformed(ActionButton.java:171)
	at com.intellij.openapi.actionSystem.impl.ActionButton.lambda$performAction$0(ActionButton.java:150)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:261)
	at com.intellij.openapi.actionSystem.impl.ActionButton.performAction(ActionButton.java:150)
	at com.intellij.openapi.actionSystem.impl.ActionButton.processMouseEvent(ActionButton.java:427)
	at java.desktop/java.awt.Component.processEvent(Component.java:6417)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5027)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4859)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2784)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4859)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:778)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:751)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:749)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:748)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:886)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.java:815)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:752)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:442)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:825)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:441)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:794)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:493)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by marhali** [comment]
My bad, try `.*.json`

**Comment by cannap** [comment]
i tried it still scans the js files maybe just dont throw an error? and use other pattern to ignore js i would make a pr but i have no idea about JAVA
java.lang.UnsupportedOperationException: Unsupported i18n locale file format: js

**Comment by marhali** [comment]
Ah okay, I see the problem... Will be fixed in the upcoming release 🙂

**Comment by marhali** [comment]
Should have been fixed now with latest release [v1.5.0](https://github.com/marhali/easy-i18n/releases/tag/v1.5.0) 😃.
